atlasaction/action.go

```diff
@@ -747,6 +818,83 @@ func (a *Actions) SchemaApply(ctx context.Context) error {
 	return nil
 }
 
+// MonitorSchema runs the Action for "ariga/atlas-action/monitor/schema"
+func (a *Actions) MonitorSchema(ctx context.Context) error {
+	db, err := url.Parse(a.GetInput("url"))
+	if err != nil {
+		return err
+	}
+	var (
+		id = cloud.ScopeIdent{
+			URL:     db.Redacted(),
+			ExtID:   a.GetInput("slug"),
+			Schemas: splitOrEmpty(a.GetInput("schemas"), ","),
+			Exclude: splitOrEmpty(a.GetInput("exclude"), ","),
+		}
+	)
+	cc, err := a.CloudClient("cloud-token")
+	if err != nil {
+		return err
+	}
+	res, err := a.Atlas.SchemaInspect(ctx, &atlasexec.SchemaInspectParams{
+		URL:     a.GetInput("url"),
+		Schema:  id.Schemas,
+		Exclude: id.Schemas,
+		Format:  `{{ printf "%s\n%s" .Hash .MarshalHCL }}`,
```

Comment:
Maybe build this as a valid HCL, mean add `# ` to mark it as comment
```suggestion
		Format:  `{{ printf "# %s\n%s" .Hash .MarshalHCL }}`,
```